### PR TITLE
[DOC-10478] Updated page topic type attributes...

### PR DESCRIPTION
**Pull request by sarahlwelton**
...on all inconsistent files in n1ql/pages.

Split transactions.adoc into two separate topics by adding...

...transactions-walkthrough.adoc.

Updated multiple.n1ql and results.jsonc to support changes in...

....transactions.adoc and transactions-walkthrough.adoc.

### CODE REVIEW

**Review by simon-dew — COMMENTED** [review]
Thanks @sarahlwelton &mdash; it will be a little while before I get a chance to look at this one.

File: modules/n1ql/examples/transactions/multiple.n1ql [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* tag::settings[] */
-\SET -txtimeout "2m"; -- <.>
-\SET -scan_consistency "not_bounded"; -- <.>
-\SET -durability_level "none"; -- <.>
+\SET -txtimeout "2m"; 
+\SET -scan_consistency "not_bounded";
+\SET -durability_level "none";
```
Comment (simon-dew) [inline modules/n1ql/examples/transactions/multiple.n1ql]: This example file is used in multiple locations and _some_ of those need the callout numbers. Please reinstate. If you need to include an example file and exclude the callouts, use the `no-callouts` role, e.g. `[source.no-callouts,sqlpp]`

```suggestion
\SET -txtimeout "2m"; -- <.>
\SET -scan_consistency "not_bounded"; -- <.>
\SET -durability_level "none"; -- <.>
```

File: modules/n1ql/examples/transactions/results.jsonc [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "_sequence_result": [
 // tag::begin[]
       {
-        "txid": "d81d9b4a-b758-4f98-b007-87ba262d3a51" // <.>
+        "txid": "d81d9b4a-b758-4f98-b007-87ba262d3a51"
```
Comment (simon-dew) [inline modules/n1ql/examples/transactions/results.jsonc]: As above, please reinstate callout numbers throughout

File: modules/n1ql/examples/transactions/results.jsonc [hunk 3]
```diff
@@ -60,26 +61,27 @@
...
   {
     "_sequence_num": 7,
     "_sequence_query": "\n\n-- Set a second savepoint\nSAVEPOINT s2;",
     "_sequence_query_status": "success",
     "_sequence_result": {
       "results": []
     }
-  },
-  {
+  }, // end::user0[]
```
Comment (simon-dew) [inline modules/n1ql/examples/transactions/results.jsonc]: Not tested this, but I don't think AsciiDoc tagging will work if the tag comment is not on a line by itself

File: modules/n1ql/pages/advanced.adoc [hunk 4]
```diff
@@ -17,7 +18,7 @@ endif::[]
 
 == Flex Indexes
 
-In Couchbase Server 6.6 Enterprise Edition and later, the Flex Index feature provides the ability for a N1QL query to use a Full Text Search index transparently with standard N1QL syntax.
+In Couchbase Server 6.6 Enterprise Edition and later, the Flex Index feature provides the ability for a {sqlpp} query to use a Full Text Search index transparently with standard {sqlpp} syntax.
```
Comment (simon-dew) [inline modules/n1ql/pages/advanced.adoc]: Good call, but please note there is a whole epic of tickets for this change as part of [DOC-10445](https://issues.couchbase.com/browse/DOC-10445). It was agreed in the planning for the N1QL→SLQ++ rebrand that we would make the switchover in the latest version, i.e. release/7.2, whereas this PR is for release/7.1.

**Review by simon-dew — CHANGES_REQUESTED** [review]
I have had a quick look and these are the first few comments that spring to mind. I'll look at the transactions walkthrough when I get the chance.

File: modules/n1ql/pages/advanced.adoc (hunk 4 above)
Comment (sarahlwelton) [reply modules/n1ql/pages/advanced.adoc]: The attribute displays as N1QL in the Antora playbook for this version so this is a change that would have 0 impact, but save time in the future.

Comment (simon-dew) [reply modules/n1ql/pages/advanced.adoc]: The impact is, it makes it harder for future writers to track changes back to the original issue that requested them, if they are fixed by a different issue. If you're going to leave this change here please can you add issue number DOC-10447 to the body of the commit message so we can track this change through this PR.

File: modules/n1ql/pages/advanced.adoc [hunk 5]
```diff
@@ -27,8 +28,9 @@ The cost-based optimizer takes into account the cost of memory, CPU, network tra
...
-== N1QL Support for Couchbase Transactions
+== {sqlpp} Support for Couchbase Transactions
 
-N1QL offers full support for Couchbase ACID transactions.
+{sqlpp} offers full support for Couchbase ACID transactions.
 
-* xref:n1ql:n1ql-language-reference/transactions.adoc[N1QL Support for Couchbase Transactions]
\ No newline at end of file
+* xref:n1ql:n1ql-language-reference/transactions.adoc[]
+* xref:n1ql:n1ql-language-reference/transactions-walkthrough.adoc[]
```
Comment (simon-dew) [inline modules/n1ql/pages/advanced.adoc]: Stash this change and use it to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)

File: modules/n1ql/pages/n1ql-language-reference/transactions-walkthrough.adoc [hunk 6]
```diff
@@ -0,0 +1,249 @@
+= Use {sqlpp} with Couchbase Transactions
```
Comment (simon-dew) [inline modules/n1ql/pages/n1ql-language-reference/transactions-walkthrough.adoc]: Stash this change and use it to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)

File: modules/n1ql/pages/n1ql-language-reference/transactions.adoc [hunk 7]
```diff
@@ -139,6 +139,8 @@ These are explained in the sections below.
 Note that some Couchbase SDKs provide APIs to support Couchbase transactions.
 For further details, refer to {transactions}[Transactions].
 
+TIP: For a walkthrough of how to create a transaction and use other transaction statements with the Query Workbench, cbq shell, or Query REST API, see xref:n1ql:n1ql-language-reference/transactions-walkthrough.adoc[].
+
```
Comment (simon-dew) [inline modules/n1ql/pages/n1ql-language-reference/transactions.adoc]: Stash this change and use it to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)

File: modules/n1ql/pages/n1ql-language-reference/transactions.adoc [hunk 8]
```diff
@@ -184,194 +186,6 @@ When developing a transaction with an SDK, the transaction may contain a mixture
...
-----
-
-.Commit the transaction
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=commit]
-----
-
-<.> After beginning the transaction, each subsequent statement in the transaction must specify the transaction ID that was generated when the transaction began.
-====
```
Comment (simon-dew) [inline modules/n1ql/pages/n1ql-language-reference/transactions.adoc]: Stash this change and use it to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)

**Review by simon-dew — CHANGES_REQUESTED** [review]
The page topic type attributes are all great 👍

Splitting the transaction walkthrough into a new file is useful and necessary but falls outside the scope of this issue. Created [DOC-10501](https://issues.couchbase.com/browse/DOC-10501) to address those changes.

File: modules/ROOT/nav.adoc [hunk 9]
```diff
@@ -89,7 +89,8 @@
  ** xref:n1ql:advanced.adoc[Advanced Features]
   *** xref:n1ql:n1ql-language-reference/flex-indexes.adoc[Flex Indexes]
   *** xref:n1ql:n1ql-language-reference/cost-based-optimizer.adoc[Cost-Based Optimizer]
-  *** xref:n1ql:n1ql-language-reference/transactions.adoc[N1QL Support for Couchbase Transactions]
+  *** xref:n1ql:n1ql-language-reference/transactions.adoc[]
+  *** xref:n1ql:n1ql-language-reference/transactions-walkthrough.adoc[]
```
Comment (simon-dew) [inline modules/ROOT/nav.adoc]: Stash this change and use it to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)

File: modules/n1ql/pages/n1ql-language-reference/transactions.adoc [hunk 10]
```diff
@@ -184,194 +186,6 @@ When developing a transaction with an SDK, the transaction may contain a mixture
 
 include::learn:partial$transaction-privileges.adoc[]
 
-== Worked Example
-
-This worked example guides you through a complete Couchbase transaction session using N1QL.
-
-[[preparation]]
```
Comment (simon-dew) [inline modules/n1ql/pages/n1ql-language-reference/transactions.adoc]: Note that removing this section breaks links in the SET TRANSACTION, SAVEPOINT, ROLLBACK TRANSACTION, COMMIT TRANSACTION and BEGIN TRANSACTION reference pages &mdash; need to fix this as part of [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)

File: modules/n1ql/examples/transactions/results.jsonc [hunk 11]
```diff
@@ -48,6 +48,7 @@
       "results": []
     }
   },
+  // tag::user0[]
```
Comment (simon-dew) [inline modules/n1ql/examples/transactions/results.jsonc]: Stash these changes and reuse them to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)

**Review by simon-dew — APPROVED** [review]
LGTM 👍

**Pull request merged**